src/VSIX/Properties/AssemblyInfo.cs

```diff
@@ -9,6 +9,7 @@
 [assembly: AssemblyTitle("NpgsqlVSIX")]
 [assembly: AssemblyDescription("")]
 [assembly: AssemblyConfiguration("")]
+[assembly: AssemblyVersion("4.1.12.4")]
```

Comment:
Is this intentionally here, is it necessary for some reason?

Reply:
Originaly there is no assembly version; defaults to 1.0.0. 

Tried to create a version based on npgsql provider used.

Reply:
Is the assembly version important here, does anyone look at it?

Reply:
No, it is not. I am removing it